fathom/src/core/semantics.rs

```diff
@@ -97,7 +146,7 @@ pub enum Head {
 #[derive(Debug, Clone)]
 pub enum Elim<'arena> {
     /// Function applications.
-    FunApp(Plicity, ArcValue<'arena>),
+    FunApp(Plicity, LazyValue<'arena>),
```

Comment:
Could also use `LazyValue` in the parameter types of `Value::FunType` and `Value::FormatCond`.